src/components/Reports/Reports.js

```diff
@@ -17,17 +17,21 @@ import { addDays } from '../../utils';
 
 const periodSelectedDaysDefault = 7;
 
+const getCurrentDate = (date) => {
+  return new Date(date.getMonth() + 1 + '/' + date.getDate() + '/' + date.getFullYear());
```

Comment:
There should be a way of doing this without using string, let me research...

Reply:
```suggestion
  return new Date(date.getFullYear(), date.getMonth(), date.getDate());
```

Reply:
changed